net/mini_snmpd/Makefile

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (C) 2009-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mini_snmpd
+PKG_VERSION:=1.4-rc1
+PKG_RELEASE:=3
+PKG_MAINTAINER:=Luke McKee <hojuruku@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/troglobit/mini-snmpd.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=203d92e60ed09466d6676c6ad20ad6cb2ce08a5d
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+PKG_FIXUP:=autoreconf
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+# As warned by upstream maintainer and binutils 
+# however compiler warnings can be ignored until next binutils release 
+# https://github.com/wongsyrone/openwrt-1/issues/67
+# PKG_SSP:=0
+# PKG_RELRO:=0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/mini_snmpd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=A tiny SNMP server for embedded systems 
+  URL:=http://troglobit.github.io/mini-snmpd.html
+# uncomment if you just want the binary, not the init script
+# openwrt requires init script runtime dependencies be defined for make menuconfig 
+# (e.g busybox sysntpd)
+  DEPENDS:=+jsonfilter +ubus +procd +ubox
+endef
+
+CONFIGURE_ARGS+= \
+	$(if $(CONFIG_IPV6),,--disable-ipv6)
+
+# Configure weirdness - Disabled by default, explicitately disabling turns feature on!
+#     --disable-debug \
+#	--disable-demo - Upstream Github Issue #4 Fixed 20160707
+
+# We make sure that only openwrt buildroot does the stripping as per user request via make menuconfig
+CONFIGURE_VARS+= LD="$(TARGET_CC)" \
+	STRIP="/bin/true" \
+	CFLAGS="$(TARGET_CFLAGS)"
+MAKE_FLAGS+= STRIP="/bin/true" 
+
+# FYI upstream suggested stripping options
+#	STRIP="strip -S --strip-unneeded --remove-section=.note.gnu.gold-version --remove-section=.comment --remove-section=.note --remove-section=.note.gnu.build-id --remove-section=.note.ABI-tag mini_snmpd" 
```

Comment:
the 8 lines (including blank and comments) doesn't include CONFIG_IPV6
for the STRIP part it's only calling sstrip (checked with strace)